src/common/constants.ts

```diff
@@ -3,40 +3,52 @@ const isBrowser = () => typeof window !== 'undefined'
 const deriv_com_url = 'deriv.com'
 const deriv_me_url = 'deriv.me'
 const deriv_be_url = 'deriv.be'
+const staging_deriv_be_url = 'staging.deriv.be'
 
 export const deriv_com_app_id = 16929
 export const deriv_me_app_id = 1411
 export const deriv_be_app_id = 30767
 
-const supported_domains = [deriv_com_url, deriv_me_url, deriv_be_url]
-const domain_url =
-    isBrowser() && supported_domains.includes(window.location.hostname)
-        ? window.location.hostname
-        : deriv_com_url
+const getDomainUrl = (): string => {
+    if (isBrowser()) {
+        switch (window.location.hostname) {
+            case deriv_com_url:
+                return deriv_com_url
+            case deriv_me_url:
+                return deriv_me_url
+            case deriv_be_url:
+                return deriv_be_url
+            case staging_deriv_be_url:
+                return deriv_be_url
+            default:
+                return deriv_com_url
+        }
+    }
+}
```

Comment:
Can we have an object with a pair of domain name and it's corresponding url, so instead of using `switch` we can just get the keys of object and if the hostname exist we just use the value of it otherwise return the `deriv_com_url`

```
 domain_urls = {
   [deriv_com_url] : deriv_com_url,
   [deriv_me_url]: deriv_me_url
   // and so on 
   }
   
return Object.keys(domain_urls).includes(window.location.hostname)) ? domain_urls[window.location.hostname] : deriv_com_url